Objects/longobject.c

```diff
@@ -630,7 +642,7 @@ PyLong_AsSize_t(PyObject *vv)
     x = 0;
     if (i < 0) {
         PyErr_SetString(PyExc_OverflowError,
-                   "can't convert negative value to size_t");
+                        "can't convert negative Python int to C size_t");
```

Comment:
Should we open a sub-issue of bpo-29833 for this?
IMHO, as this is very similar to bpo-29834, maybe we should just mention in bpo-29834 that in case bpo-29834 is accepted and fixed, then also PyLong_AsSize_t should also be changed in a similar way.